### ISSUE TITLE
Get-OVBaseline throws error when HPE OneView has the latest SPP (2021.04.0)

### ISSUE THREAD

**Issue by erikgraa** [HewlettPackard/POSH-HPOneView#569]
### Expected Behavior

The cmdlet

```PowerShell
Get-OVBaseline
```

should return baseline(s).

### Actual Behavior

The cmdlet

```PowerShell
Get-OVBaseline
```

returns the following:

```PowerShell
Get-OVBaseline: Cannot convert the JSON string because it contains keys with different casing. 
Please use the -AsHashTable switch instead. 
The key that was attempted to be added to the existing key 'HPE Ethernet 100Gb 2-Port QSFP56 MCX623106AS-CDAT Adapter' was 'HPE Ethernet 100Gb 2-port QSFP56 MCX623106AS-CDAT Adapter'.
```

### Steps to reproduce

1. Add the latest Gen10 SPP (2021.04.0) to HPE OneView 6.00
2. Run the cmdlet 

```PowerShell
Get-OVBaseline
```

### Version Information
HPE OneView PowerShell Library Version (`Get-OVVersion` or `$PSLibraryVersion`): 6.0.2719.1668
HPE OneView Appliance Version (`Get-OVVersion -ApplianceVer`):  6.00.00.432224.00
Output from `$PSVersionTable` on your Windows Host:

```PowerShell
PSVersion                      7.1.3
PSEdition                      Core
GitCommitId                    7.1.3
OS                             Microsoft Windows 10.0.17763
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by ChrisLynchHPE** [comment]
Can you provide the output from `Get-OVCommandTrace -ScriptBlock { Get-OVBaseline }`, and provide the generated file?

**Comment by erikgraa** [comment]
Attached is the output from ```Get-OVCommandTrace -ScriptBlock { Get-OVBaseline }```

[2105010140_HPEOV_transcript.log](https://github.com/HewlettPackard/POSH-HPEOneView/files/6408508/2105010140_HPEOV_transcript.log)

**Comment by ChrisLynchHPE** [comment]
Thanks.  I'm refactoring the logic on identifying this issue.  I'm trying to wrap up an update to the 6.0 library here, along with getting a 6.1 library out (for the just released OneView 6.1 version.)

**Comment by ChrisLynchHPE** [comment]
Fixes were committed and released to help address this issue.  Closing due to no further activity.  This issue can always be re-opened if needed.
